src/azure-cli/setup.py

```diff
@@ -149,7 +149,7 @@
     'semver==2.13.0',
     'six>=1.10.0',  # six is still used by countless extensions
     'sshtunnel~=0.1.4',
-    'urllib3[secure]',
+    'urllib3',
```

Comment:
You may want to retain the dependencies that were originally installed via `urllib3[secure]` here like `idna, pyOpenSSL, etc. That it up for the Azure CLI team to decide.

Reply:
May I ask why you want to modify this?